src/main/java/com/werfen/report/model/GridReportField.java

```diff
@@ -0,0 +1,38 @@
+package com.werfen.report.model;
+
+import com.werfen.report.util.GeneralConfiguration;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+
+@NoArgsConstructor
+@Getter
+public class GridReportField {
+    private String name;
+    private String value;
+    private String defaultNullValue;
+
+    public static GridReportField of(String name, String value) {
+        GridReportField gridReportField =  new GridReportField();
```

Comment:
After better thinking on this, couldn't simplify this by calling the implementation of the of method with 3 params? Something like this, just to reduce some redundant lines...

`
public static GridReportField of(String name, String value) {
    return GridReportField.of(name, value, GeneralConfiguration.getDefaultNullString());
}
`

Reply:
Done